cmd/gdeploy/commands/create.go

```diff
@@ -42,17 +42,23 @@ var CreateCommand = cli.Command{
 		if err != nil {
 			return err
 		}
-		o.PrintTable()
 
-		clio.Success("Your Granted deployment has been created")
-		clio.Info(`Here are your next steps to get started:
+		if status == "CREATE_COMPLETE" {
+			clio.Success("Your Granted deployment has been created")
+			o.PrintTable()
+
+			clio.Info(`Here are your next steps to get started:
 
   1) create an admin user so you can log in: 'gdeploy users create --admin -u YOUR_EMAIL_ADDRESS'
   2) add an Access Provider: 'gdeploy provider add'
   3) visit the web dashboard: 'gdeploy dashboard open'
 
 Check out the next steps in our getting started guide for more information: https://docs.comonfate.io/granted-approvals/getting-started/deploying
 `)
+		} else {
+			clio.Warn("Your Granted deployment update ended in status %s", status)
```

Comment:
Word this specific to creating
```suggestion
			clio.Warn("Creating your Granted deployment failed with a final status %s", status)
```